### DISCUSSION TITLE
Is there a way to automatically update ExplorerPatcher?

### DISCUSSION THREAD

**Discussion by 75338** [valinet/ExplorerPatcher#2286] [Q&A]
If there is, I'm not seeing it. I'm guessing this is due to security concerns by the developers. As is, I just updated Windows 11 and it ofc radically effected my taskbar etc.

Couldn't find ExplorerPatcher in the list of apps in the Start Menu. Right-click the taskbar as the "Updated ExplorerPatcher notification" says -  just brought the standard Win 11 settings. Couldn't find ExplorerPatcher in the list of programs installed. Finally found it by searching in File Explorer after forgetting what the program was called.

I'd recommend making the update notification at least include a mention of what to do if right-clicking the taskbar doesn't result in a "Properties" option showing up, as this is rather confusing to a not super tech savvy person.

Any help is appreciated. Thank you.

**Comment by tallman69** [comment]
You did not state whether or not you were able to open the ExplorerPatcher Properties app.  If not, start typing `properties (explorerpatcher)` into the start menu or Windows search box.  Click on the Properties (ExplorerPatcher) search result, and the Properties app will open.

**Comment by pyrates999** [comment]
to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

Then go to advanced and make sure the option **Do not change the taskbar context menu (e.g. do not display the "Properties"** is not enabled. Then you can right click on the taskbar and you should see the **Properties** option.
